ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/utils/StringUtils.java

```diff
@@ -60,6 +67,57 @@ public static String shortenedString(Object input) {
         return value;
     }
 
+    /**
+     * Highlight and show the error position.
+     * The highlighted text would follow format,
+     * <pre>
+     * incompatible types: expected 'string', found 'int'
+     *     int i = "Hello";
+     *             ^-----^
+     * </pre>
+     * However if the error is multiline, it would follow following format,
+     * <pre>
+     *
```

Comment:
Are you missing some content here?

Reply:
It's supposed to say `However if the error is multiline, it would not highlight the error.` I'll fix this.